README.md

```diff
@@ -86,3 +71,40 @@ Open Hootsuite https://hootsuite.com/dashboard
 
 Click on the App Directory in the left hand nav.  Under Developers -> My Apps
 install your new App, adding to a new stream.
+
+## Getting your FullContact API Key
+
+Go to https://fullcontact.com and click "Start Building Now".
+
+Create an account with FullContact.
+
+When on the dashboard, you will see your API key in the lower section of the page.
+
+## Creating Config File
+
+Install config through npm
+````
+npm install config
+````
+
+Make a folder for your config files
+````
+mkdir config
+````
+
+create the development version:
```

Comment:
This file should be created by default with the necessary parameter with no value.